docs/usage.rst

```diff
@@ -1,22 +1,78 @@
 Usage Examples
 ==============
 
-Encoding & Decoding Tokens
+Encoding & Decoding HS256 Tokens
 ---------------------------------
 
 .. code-block:: python
 
-    >>import jwt
-    >>encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
+    >> import jwt
+    >> encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
+    >> encoded
     'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzb21lIjoicGF5bG9hZCJ9.4twFt5NiznN84AWoo1d7KO1T_yoc0Z6XOpOVswacPZg'
+    >> jwt.decode(encoded, 'secret', algorithms=['HS256'])
+    {u'some': u'payload'}
+
+
+Encoding & Decoding RS256 Tokens
+---------------------------------
+
+.. code-block:: python
+
+    >> import jwt
+    >> from cryptography.hazmat.backends import default_backend
+    >> from cryptography.hazmat.primitives import serialization
+    >> # openssl genrsa 1024 | tee private_key.pem
```

Comment:
Recommending people use 1024-bit keys is not a good idea since they are known to be weak now.